### PR TITLE
Added a note about Sodium PHP extension

### PR DESCRIPTION
**Pull request by theHocineSaad**
Sodium PHP extension is required by https://github.com/box-project/box

https://github.com/laravel-zero/laravel-zero/issues/466

### CODE REVIEW

File: installation.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ description: Create a new Laravel Zero project
 
 # Installation
 
-> **Requires [PHP 8.1+](https://php.net/releases)**
+> **Requires:**
+>- [PHP 8.1+](https://php.net/releases).
+>- `sodium`  PHP extension (required by box-project/box - the application bundler we use).
```
Comment (owenvoke) [inline installation.md]:
```suggestion
>- `sodium` PHP extension (required by [Box](https://github.com/box-project/box) - the application bundler we use).
```

**Review by owenvoke — CHANGES_REQUESTED** [review]
Hi @theHocineSaad, thanks for the PR. Just a small change to the formatting, but otherwise looks great!

**Comment by theHocineSaad** [comment]
@owenvoke I added the requested change.

**Review by owenvoke — APPROVED** [review]

**Comment by owenvoke** [comment]
Thank you @theHocineSaad! 🙌🏻

**Pull request merged**